src/main/scala/xiangshan/frontend/NewFtq.scala

```diff
@@ -24,6 +24,12 @@ import xiangshan._
 import xiangshan.frontend.icache._
 import xiangshan.backend.CtrlToFtqIO
 import xiangshan.backend.decode.ImmUnion
+import huancun.utils.ChiselDB
+
+class FtqDebugBundle extends Bundle {
```

Comment:
could add more information, such as branch type, whether mispredicted, taken or not, BPU stage the prediction from